prboom2/src/dsda/configuration.c

```diff
@@ -756,6 +756,10 @@ dsda_config_t dsda_config[dsda_config_count] = {
     "mus_opl_gain", dsda_config_mus_opl_gain,
     dsda_config_int, 0, 1000, { 50 }
   },
+  [dsda_config_mus_opl_opl3mode] = {
+    "mus_opl_opl3mode", dsda_config_mus_opl_opl3mode,
```

Comment:
Why would someone want to disable this? What is the difference exactly?

Reply:
OPL2 sounds worse and only mono. But most of the popular sound cards in the early 90s were OPL2.

Reply:
I see, so it's there for people who want the crustiest option :^)